### ISSUE TITLE
Is this module working on v11 ?

### ISSUE THREAD

**Issue by uniwars** [sdenec/tidy-ui_game-settings#99]
Hi, I used a script to see compatibility checks for all the modules I use before updating to V11 and it returned that this module is not compatible with v11. Is it a compatibility update issue or is the module broken in v11 ? Thank you.

**Comment by karpana** [comment]
@uniwars 

There is at lesat one issue (such as #98) with this module on v11.

But in the absence of a formal compatibiliby confirmation from the author, I can provide some anectodal input/feedback that it does seem to be working properly on v11, even if there are some visual quirks.

**Comment by SLinxTheFox** [comment]
Yes, but you cannot search in your module settings with it active, see #91
